code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -196,6 +198,7 @@
 		holding = null
 	if(new_tank)
 		holding = new_tank
+		playsound(src, pick(insert_sound), sound_vol)
```

Comment:
It is criminal that there's no removal sound, it would be SO satisfying you're killing me